test/unit/sidekiq_mover_test.rb

```diff
@@ -67,14 +67,58 @@ def test_releases_locks_in_source_dc
     assert_equal [1, 2], JobHelper.job_result_queue.map {|arg| arg[0]['num']}
   end
 
+  def test_can_move_retry_queue
+    CrashingJob.perform_async(num: 1)
```

Comment:
can we test with > 1 . to ensure _all_ jobs are moved

Reply:
I've adjusted the test to push two jobs.

Reply:
Tho I don't think that's important to check since it's tested more directly in other tests.

Reply:
What really matters is that when we specify the queue `retry` that we actually get the retry queue.